covid-vaccine-faq-bot/assets/client.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+let phoneNumber;
+let flowSid;
+
+// eslint-disable-next-line no-restricted-globals
+const baseUrl = new URL(location.href);
+baseUrl.pathname = baseUrl
+  .pathname
+  .replace(/\/index\.html$/, '');
+delete baseUrl.hash;
+delete baseUrl.search;
+const fullUrl = baseUrl
+  .href
+  .substr(0, baseUrl.href.length - 1);
+
+fetch(`${fullUrl}/return-config`)
+  .then((response) => response.json())
+  .then((json) => {
+    phoneNumber = json.phone_number;
+    // Grab the phone number being used and display it to help the user test their app
+    const phoneNumberElements = $('.phone-number');
+    phoneNumberElements.html(phoneNumber);
+  });
```

Comment:
Catch errors here and at a minimum let the user know that there was a problem in the UI.